docs/units/findable-data/topics/3-metadata-standards.md

```diff
@@ -1,12 +1,13 @@
 ---
 number: 2.3
-title: Metadata standards
+title: Metadata standards for findability
 status: ready_for_review
 template: 'lesson-plan.html'
 authors:
   - "0000-0003-1378-5495"
   - "0000-0002-9016-4820"
   - "0000-0001-6989-2252"
+  - "0000-0003-0771-3516"
```

Comment:
Add contributor info to the contributors' table (https://github.com/elixir-europe-training/ELIXIR-TrP-FAIR-Converge/blob/main/_data/contributors.yaml) as part of this PR